packages/runtime-dom/src/apiCustomElement.ts

```diff
@@ -258,7 +258,9 @@ export class VueElement extends BaseClass {
 
     const asyncDef = (this._def as ComponentOptions).__asyncLoader
     if (asyncDef) {
-      asyncDef().then(resolve)
+      asyncDef()
+        .then(resolve)
+        .then(() => this._update())
```

Comment:
This is necessary to have the asyncWrapper'S props updated before it renders the nested actual component.